tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -105,6 +105,13 @@ BusType = PCIe
 PCI.BusNum = 0x0
 PCI.DevNum = 0x0
 
+[DAQmxDevice nidaqmxUSBTester]
+ProductType = USB-9265
```

Comment:
The USB-9265 is EOL. NI-DAQmx tends to support discontinued products for several years, but it's better for new tests to use products that are not discontinued.

Suggestion: replace this with a PXIe-4322 and test one of these pairs of properties:       
- OpenCurrentLoopChans / OpenCurrentLoopChansExist
- OvercurrentChans / OvercurrentChansExist

Reply:
I used `OpenCurrentLoopChans` / `OpenCurrentLoopChansExist` for boolean & string list respectively and tests passed as expected.